scripts/doc-checker.py

```diff
@@ -151,11 +249,15 @@ def pydocstyle_validate(path: pathlib.Path, add_ignore: List[str]) -> int:
 
 
 def monkeypatching():
-    """Monkeypatch decorators which incorrectly define __doc__ attribute."""
+    """Monkeypatch decorators which change __doc__ attribute."""
     import ray
+    import modin.utils
 
     ray.remote = lambda *args, **kwargs: args[0]
 
+    modin.utils._inherit_docstrings = lambda *args, **kwargs: lambda cls: cls
+    modin.utils._inherit_func_docstring = lambda *args, **kwargs: lambda func: func
```

Comment:
I don't understand this change. Why are you turning off these decorators? `ray.remote` has a bug which those do not.
In my opinion skipping the checks if `# noqa` directive is given should happen even with decorators enabled (but we may need to add more `noqa` values in there).

Reply:
I turned them off because:
1) we do not need to do what will not be used in the future (by not doing, we save time)
2) we check a lot of errors that Pandas does not check, therefore we will need to add a lot of error codes that can be ignored (this will complicate both support and readability of the code)

Reply:
Do note that I eventually plan to use those decorators on functions where we _do_ have our own docstrings.
It would append a link to Pandas API, but we still want to check the docstring.

How many error checks we need to ignore if we do not turn off these?..

Reply:
>> Added next codes: GL08, SS03, PR01, PR02, PR08, PR09 and GL01, GL02

Any of these errors may occur.

Reply:
>>Do note that I eventually plan to use those decorators on functions where we do have our own docstrings.
>>It would append a link to Pandas API, but we still want to check the docstring.

It turns out that your decorator can work in two modes - adding the entire documentation or just a link, respectively, based on some parameter, I can decide when to disable this decorator, and when not